ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -996,6 +1025,10 @@ + (BOOL)supportsSecureCoding {
     return YES;
 }
 
+- (NSString*)stringForAnswer:(id)answer {
+    return ORKTimeOfDayStringFromComponents(answer);
+}
```

Comment:
Yep, you are right. We've already decided to open a separate issue on date formatters after the review step itself is finished.